src/fstests/runner.py

```diff
@@ -55,7 +55,8 @@ def _schedule_job(self, tarball_node, device_config, tmp):
             params.update(self._plan.params)
             params.update(device_config.params)
             templates = ['config/runtime',
-                         '/etc/kernelci/runtime']
+                         '/etc/kernelci/runtime',
+                         'src/fstests']
```

Comment:
This is meant to go in the 2nd commit as in the last iteration...

Please, make sure you double check your commits individually before pushing and also be more careful during the rebase process.

Reply:
Ok, Sorry for this mistake. I checked this carefully (double check)  and verified after pushing to github again that each commit represents what is said in the message.